modules/webp-default/class-webp-default.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * WebP-Default Module
+ *
+ * @package performance-lab
+ * @since 1.0.0
+ */
+
+/**
+ * WebP-Default Module
+ *
+ * Switches the default sub sized output format for images to WebP when supported.
+ *
+ * @since 1.0.0
+ */
+class WebP_Default {
```

Comment:
Nit-pick, but since we're targeting WordPress core, for a simple filter application like this I don't think we need a class. Two simple functions would do - you could even put them right into the `load.php` file.

Reply:
I wasn't sure about this. parts of core as written as classes, parts are purely functional. I agree this is a bit overkill here and was maybe trying to set a precedent for modules.  I'll simplify into a single file and remove the class.